components/mpas-ocean/src/Registry.xml

```diff
@@ -3227,6 +3243,88 @@
 			 description="Wind speed at 10 meter."
 		/>
 
+                <!-- Waves coupling fields -->
+		<var name="stokesDriftZonal" type="real" dimensions="nVertLevels nCells Time" units="m s^{-1}"
+			 description="Zonal component of the wave-induced Stokes drift current."
+			 packages="activeWavePKG"
+		/>
+		<var name="stokesDriftMeridional" type="real" dimensions="nVertLevels nCells Time" units="m s^{-1}"
+			 description="Meridional component of the wave-induced Stokes drift current."
+			 packages="activeWavePKG"
+		/>
+		<var name="stokesDriftSurfaceZonal" type="real" dimensions="nCells Time" units="m s^{-1}"
+			 description="Zonal component of the wave-induced Stokes surface drift current."
+			 packages="activeWavePKG"
+		/>
+		<var name="stokesDriftSurfaceMeridional" type="real" dimensions="nCells Time" units="m s^{-1}"
+			 description="Meridional component of the wave-induced Stokes surface drift current."
+			 packages="activeWavePKG"
+		/>
+		<var name="significantWaveHeight" type="real" dimensions="nCells Time" units="m"
+			 description="Significant wave height is the average of the highest 1/3 of the waves"
+			 packages="activeWavePKG"
+		/>
+		<var name="peakWaveFrequency" type="real" dimensions="nCells Time" units="s^{-1}"
+			 description="Frequency of peak of wave spectrum"
+			 packages="activeWavePKG"
+		/>
+		<var name="peakWaveDirection" type="real" dimensions="nCells Time" units="deg"
+			 description="Direction of peak of wave spectrum"
+			 packages="activeWavePKG"
+		/>
+		<var name="langmuirNumber" type="real" dimensions="nCells Time" units="m"
+			 description="Projected surface layer averaged Langmuir number from Van Roekel et al. 2012."
+		/>
+		<var name="stokesDriftZonalWavenumber1" type="real" dimensions="nCells Time" units="m s^{-1}"
+			 description="Zonal component of the partitioned wave-induced Stokes drift current. Wavenumbers 1-6 are used to recontruct the full depth-dependent Stokes drift profile"
+			 packages="activeWavePKG"
+		/>
+		<var name="stokesDriftMeridionalWavenumber1" type="real" dimensions="nCells Time" units="m s^{-1}"
+			 description="Meridional component of the partitioned wave-induced Stokes drift current. Wavenumbers 1-6 are used to recontruct the full depth-dependent Stokes drift profile"
+			 packages="activeWavePKG"
+		/>
```

Comment:
I agree. I originally did this because most of the coupling variables are 1D. However, now I see that there are a couple 2D ones. I'll make this change.